ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json

```diff
@@ -2291,21 +2298,35 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "hasHearingsToVacate",
+    "CaseFieldID": "hasPastHearings",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "hasFutureHearings",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "hasFutureHearingDateFlag",
+    "CaseFieldID": "hasHearingsToVacate",
```

Comment:
@SomeshRA is this backward compatible? i think we spoke about this before and it's ok but want to be sure...